src/Analysis/Ast/Impl/Typeshed/stdlib/3.4/pathlib.pyi

```diff
@@ -8,7 +8,7 @@ import sys
 _P = TypeVar('_P', bound='PurePath')
 
 if sys.version_info >= (3, 6):
-    _PurePathBase = os.PathLike[str]
+    _PurePathBase = os.PathLike
```

Comment:
Fixing a syntax error, seems a little strange though so I might be wrong.

Reply:
Also it's a little weird that they use the type parameter P without declaring the class is `Generic[P]` but I'm not sure if that is a syntax error.

Reply:
Not our stuff.

Reply:
Don't change it, we take snapshots from typeshed repo.